etc/static-hi-puma/index.html

```diff
@@ -0,0 +1 @@
+<html><h1>index.html</h1></html>
```

Comment:
this should never show up. it's included here to verify that we weren't able to look outside the `public` directory.